<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CBattle</class>
 <widget class="QDialog" name="CBattle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1001</width>
    <height>998</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="headlineLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>641</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Kampf um</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>180</y>
     <width>131</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="attackerInfoLayout">
    <item>
     <widget class="QLabel" name="attackerUnitsTotal">
      <property name="text">
       <string>Einheiten gesamt:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="attackerUnitsActive">
      <property name="text">
       <string>davon noch aktiv:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="attackerUnitsSuppressed">
      <property name="text">
       <string>davon unterdrückt:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="attackerUnitsLost">
      <property name="text">
       <string>davon gefallen:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="attackerLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string> Angreifer Gesamt</string>
   </property>
  </widget>
  <widget class="QLabel" name="defenderLabel">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>110</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Verteidiger Gesamt</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>180</y>
     <width>41</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="attackerValuesLayout">
    <item>
     <widget class="QLabel" name="attackerUnitsTotalValue">
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="attackerUnitsAcitveValue">
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="attackerUnitsSuppressedValue">
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="attackerUnitsLostValue">
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>180</y>
     <width>131</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="defenerInfoLayout">
    <item>
     <widget class="QLabel" name="defenderUnitsTotal">
      <property name="text">
       <string>Einheiten gesamt:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="defenderUnitsActive">
      <property name="text">
       <string>davon noch aktiv:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="defenderUnitsSuppressed">
      <property name="text">
       <string>davon unterdrückt:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="defenderUnitsLost">
      <property name="text">
       <string>davon gefallen:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>180</y>
     <width>41</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="defenerValuesLayout">
    <item>
     <widget class="QLabel" name="defenderUnitsTotalValue">
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="defenderUnitsActiveValue">
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="defenderUnitsSuppressedValue">
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="defenderUnitsLostValue">
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>620</y>
     <width>801</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="diceLayout">
    <item>
     <layout class="QHBoxLayout" name="attackerDiceLayout"/>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="defenderDiceLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="attackerDiceLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>580</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Angriffswürfel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="defenderDiceLabel">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>580</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Verteidigerwürfel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="retreatPB">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>430</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Rückzug!!</string>
   </property>
  </widget>
  <widget class="QLabel" name="battleLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>490</y>
     <width>441</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string>Gefechtsauswertung</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>370</y>
     <width>561</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="battlePBLayout">
    <item>
     <widget class="QPushButton" name="reconAttackPB">
      <property name="text">
       <string>Hinterhalt</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="normalAttackPB">
      <property name="text">
       <string>Angriff</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="assaultAttackPB">
      <property name="text">
       <string>Sturmangriff</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>730</y>
     <width>131</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="attackerBattleInfoLayout">
    <item>
     <widget class="QLabel" name="attackerBattleUnitsTotal">
      <property name="text">
       <string>Einheiten gesamt:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="attackerBattleUnitsActive">
      <property name="text">
       <string>davon noch aktiv:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="attackerBattleUnitsSuppressed">
      <property name="text">
       <string>davon unterdrückt:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="attackerBattleUnitsLost">
      <property name="text">
       <string>davon gefallen:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>730</y>
     <width>41</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="attackerBattleValuesLayout">
    <item>
     <widget class="QLabel" name="attackerWaveTotalValue">
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="attackerWaveAcitveValue">
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="attackerWaveSuppressedValue">
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="attackerWaveLostValue">
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_7">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>730</y>
     <width>131</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="defenderBattleInfoLayout">
    <item>
     <widget class="QLabel" name="defenderBattleUnitsTotal">
      <property name="text">
       <string>Einheiten gesamt:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="defenderBattleUnitsActive">
      <property name="text">
       <string>davon noch aktiv:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="defenderBattleUnitsSuppressed">
      <property name="text">
       <string>davon unterdrückt:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="defenderBattleUnitsLost">
      <property name="text">
       <string>davon gefallen:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_8">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>730</y>
     <width>41</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="defenderBattleValuesLayout">
    <item>
     <widget class="QLabel" name="defenderWaveTotalValue">
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="defenderWaveAcitveValue">
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="defenderWaveSuppressedValue">
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="defenderWaveLostValue">
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="battleAcceptPB">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>900</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>...das musste in Kauf genommen werden...</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>340</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="waveLayout">
    <item>
     <widget class="QLabel" name="WaveLabel">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Angriffswelle (max. 5):</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="waveValue">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>560</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="bonusLayout">
    <item>
     <widget class="QLabel" name="bonusLabel">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Würfelmultiplikator:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="bonusValue">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
